view/view_manager.py

```
@@ -0,0 +1,38 @@
+        self.__init_menu_frame()
+
+    def __init_main_window(self) -> None:
+        self.tkWindow = Window("app title")
+
+    def __init_menu_frame(self) -> None:
+        self.menu_frame = Menu(self)
+
+    def start_application(self) -> None:
+        self.tkWindow.tk.mainloop()
+
+    def update_view_frame(self, frame: ViewFrame) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)